### PR TITLE
Adding law

### PR DESCRIPTION
**Pull request by riga**
This PR adds the law (luigi analysis workflows) package, starting from version 0.0.41, to the index.

- [GitHub](https://github.com/riga/law)
- [PyPI](https://pypi.org/project/law/)

### Checklist

- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] ~If static libraries are linked in, the license of the static library is packaged.~ Does not apply.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/law/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+    - six >=1.10
+    - luigi >=2.8.2,<3  # [py<=27]
```
Comment (scopatz) [inline recipes/law/meta.yaml:22]: Is this actually being used on 2.7?

File: recipes/law/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3b6e457bb99da69ddab03e91a80057958ee9477dba7bb6f8823059b2cbce19b3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```
Comment (scopatz) [inline recipes/law/meta.yaml:15]: I think this could be a noarch python package

Comment (riga) [reply recipes/law/meta.yaml:15]: Ok, will change that ✔︎.

File: recipes/law/meta.yaml (hunk 1 above)
Comment (riga) [reply recipes/law/meta.yaml:22]: Sadly, yes. Parts of the physics research community is still relying on 2.7 ..

File: recipes/law/meta.yaml (hunk 2 above)
Comment (riga) [reply recipes/law/meta.yaml]: @scopatz  This change resulted in `noarch packages can't have selectors` ...
So does having selectors for dependencies rule out the possibility of making this a noarch python package?

Comment (scopatz) [reply recipes/law/meta.yaml]: Yes, it does. But I don't think you need selectors in this recipe

Comment (riga) [reply recipes/law/meta.yaml]: How can I enforce using a requirement below a certain version, depending on the python version? In this case `luigi`, which dropped python 2 support from 3.0 onwards (thus the `>=2.8.2,<3` pin).

Comment (scopatz) [reply recipes/law/meta.yaml]: Hmm yeah, there seems to be a bug in the luigi-feedstock here.  Maybe you do need selectors here right now

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**